xilem/src/view/async_repeat.rs

```diff
@@ -8,6 +8,17 @@ use xilem_core::{DynMessage, Message, MessageProxy, NoElement, View, ViewId, Vie
 
 use crate::ViewCtx;
 
+/// Launch a task which will run until the view is no longer in the tree.
+/// `future_future` is given a [`MessageProxy`], which it will store in the future it returns.
+/// This `MessageProxy` can be used to send a message to `on_event`, which can then update
+/// the app's state.
+///
+/// For exampe, this can be used with the time functions in [`crate::tokio::time`].
+///
+/// Note that this task will not be updated if the view is rebuilt, so `future_future`
+/// cannot capture.
+// TODO: More thorough documentation.
+/// See [run_once](crate::core::run_once) for details.
 pub fn async_repeat<M, F, H, State, Action, Fut>(
```

Comment:
Regarding the name, maybe this:
```suggestion
pub fn relay<M, F, H, State, Action, Fut>(
```
or `spam`^^ or maybe something like `emit_proxy`.

Hmm difficult to find a good name for this...

Reply:
I do agree that this name is not ideal.
I want to avoid it being something too abstract.

Reply:
I'm going to merge and we follow-up with a rename if needed